### PR TITLE
Feature/lt 02/cookie

### PR DESCRIPTION
**Pull request by denbit**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,30 +4,15 @@ import {LanguageList} from './Languages';
...
 import Main from './Main'
 import Footer from "./Footer";
+import Blank from "./Blank"
+import * as Cookies from "js-cookie";
 
 export const Language = React.createContext({filler: {}, slides: {}});
 export const menulist = new Array("UA", "PL", "EN", "RU");
 export const title_text = "Chemistry CODE";
-const LanguageCache = {};
+export const LanguageCache = {};
```
Comment (denbit) [inline src/App.js]: why should we export it?

File: src/App.js [hunk 2]
```diff
@@ -43,8 +28,13 @@ class App extends Component {
...
 	async componentWillMount() {
-		await this.loadLangs();
-		LanguageCache.ua = this.state.language;
+        let lang = Cookies.get('lang');
+			if(!lang){
+                lang = "ua";
+			}
+		await this.loadLangs(lang);
+
+            LanguageCache[lang] = this.state.language;
```
Comment (denbit) [inline src/App.js]: please format according to code style

File: src/App.scss [hunk 3]
```diff
@@ -87,7 +87,7 @@ section.body {
 
 
   > nav.menu {
-    width: 100%;
+    width: auto;
```
Comment (denbit) [inline src/App.scss:90]: is it related to cookies implementation? or from other branch

**Review by denbit — COMMENTED** [review]
please make changes

File: src/App.scss (hunk 3 above)
Comment (ElikaFilin) [reply src/App.scss:90]: I have fixed nav menu

**Pull request merged**